docs/source/smartdevice.rst

```diff
@@ -67,6 +67,17 @@ Refer to device type specific classes for more examples:
 :class:`SmartPlug`, :class:`SmartBulb`, :class:`SmartStrip`,
 :class:`SmartDimmer`, :class:`SmartLightStrip`.
 
+DeviceConfig class
+******************
+
+The :class:`DeviceConfig` class can be used to initialise devices with parameters to allow them to be connected to without using
+discovery.  This is required for newer KASA and TAPO devices that use different protocols for communication and will not respond
```

Comment:
```suggestion
discovery.
This is required for newer KASA and TAPO devices that use different protocols for communication and will not respond
```
Please start each sentence from a new line, this will make diffs smaller and easier to read if a single typo or something similar gets fixed.

Reply:
Done